.github/workflows/workflow.yaml

```
@@ -0,0 +1,211 @@
+ name: 'IaC Workflow | Terraform'
```

actionlint (1.7.7)

"on" section is missing in workflow

(syntax-check)